src/UI/Settings/Profile/Edit/EditProfileItemView.js

```diff
@@ -1,5 +1,21 @@
 var Marionette = require('marionette');
 
 module.exports = Marionette.ItemView.extend({
-    template : 'Settings/Profile/Edit/EditProfileItemViewTemplate'
-});
\ No newline at end of file
+    template : 'Settings/Profile/Edit/EditProfileItemViewTemplate',
+    ui: {
+        cutoff: "input[name=cutoff]"
+    },
+    onRender: function() {
+        if (this.model.get("quality").id === this.collectionListView.model.get("cutoff").id) {
+            this.ui.cutoff.prop("checked", true);
+        }
+    },
+    events : {
+        'click .cutoff': function($event) {
+            this.collectionListView.model.set("cutoff", this.model.get("quality"));
+
+            // Do not let event bubble, as the parent will register the click as select / deselect
```

Comment:
This is one thing I don't like about this change, if you're off the mark you toggle the quality off completely.